pkgs/ffigen/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
   abstract classes. Since Dart enums cannot be empty, empty native enums are
   generated as empty sealed classes. Native enum members with duplicate integer
   values are handled properly, and are equal to each other in Dart as well.
+- __Breaking change__: ffigen now emits code that uses the Dart enums in return
```

Comment:
This should probably be a single breaking change w.r.t. enums instead of two bullet points.